org.freedesktop.Sdk.Extension.node10.yaml

```
@@ -0,0 +1,45 @@
+app-id: org.freedesktop.Sdk.Extension.node10
+branch: '18.08'
+runtime: org.freedesktop.Sdk
+runtime-version: '18.08'
+sdk: org.freedesktop.Sdk
+build-extension: true
+separate-locales: false
+appstream-compose: false
+build-options:
```

Strip the binaries, otherwise we'll have a ~700Mb flatpak.
```suggestion
build-options:
  strip: true
```